core-galleon-pack/src/main/resources/packages/product.conf/pm/wildfly/resources/modules/system/layers/base/org/jboss/as/product/wildfly-core/dir/META-INF/MANIFEST.MF

```diff
@@ -1,2 +1,11 @@
 JBoss-Project-Release-Name: ${product.release.name}
 JBoss-Product-Release-Version: ${project.version}
+JBoss-Project-Banner: \n
```

Comment:
I like the idea but I'm not fond of that font (a bit too high) :) 
TGIF, I played with http://patorjk.com/software/taag/#p=testall&f=Ogre&t=WildFly%20Core and I like these ones

```
 _    _  ____  __    ____  ____  __   _  _     ___  _____  ____  ____ 
( \/\/ )(_  _)(  )  (  _ \( ___)(  ) ( \/ )   / __)(  _  )(  _ \( ___)
 )    (  _)(_  )(__  )(_) ))__)  )(__ \  /   ( (__  )(_)(  )   / )__) 
(__/\__)(____)(____)(____/(__)  (____)(__)    \___)(_____)(_)\_)(____)

 _  _  __  __    ____  ____  __    _  _     ___  __  ____  ____ 
/ )( \(  )(  )  (    \(  __)(  )  ( \/ )   / __)/  \(  _ \(  __)
\ /\ / )( / (_/\ ) D ( ) _) / (_/\ )  /   ( (__(  O ))   / ) _) 
(_/\_)(__)\____/(____/(__)  \____/(__/     \___)\__/(__\_)(____)

 __    __ _ _     _   ___ _           ___               
/ / /\ \ (_) | __| | / __\ |_   _    / __\___  _ __ ___ 
\ \/  \/ / | |/ _` |/ _\ | | | | |  / /  / _ \| '__/ _ \
 \  /\  /| | | (_| / /   | | |_| | / /__| (_) | | |  __/
  \/  \/ |_|_|\__,_\/    |_|\__, | \____/\___/|_|  \___|
                            |___/                                              
  _      ___ __   ________       _____            
 | | /| / (_) /__/ / __/ /_ __  / ___/__  _______ 
 | |/ |/ / / / _  / _// / // / / /__/ _ \/ __/ -_)
 |__/|__/_/_/\_,_/_/ /_/\_, /  \___/\___/_/  \__/ 
                       /___/                      

```